hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -583,7 +583,7 @@
     }
   ]
 
-  // Memories (ROM, RAM, eFlash) are defined at the top.
+  // Memories (RAM, eFlash) are defined at the top.
```

Comment:
I spun this last commit out into a separate change:
https://github.com/lowRISC/opentitan/pull/5871